stellargraph/mapper/full_batch_generators.py

```diff
@@ -160,6 +160,8 @@ def flow(self, node_ids, targets=None):
                 (e.g., training, validation, or test set nodes)
             targets: a 1D or 2D array of numeric node targets with shape ``(len(node_ids),)``
                 or ``(len(node_ids), target_size)``
+            use_ilocs (bool): if Ture, node_ids are represented by ilocs,
```

Comment:
The spellcheck step is highlighting the "Ture" here (and in the `flow` below):

```suggestion
            use_ilocs (bool): if True, node_ids are represented by ilocs,
```

Reply:
Thanks @huonw , to make the `FullBatchLinkGenerator` able to handle the node ilocs generated by the `UnsupervisedSampler`, I add an argument `use_ilocs` to the `FullBatchLinkGenerator.flow` function.